lib/movable_ink/aws/metadata.rb

```diff
@@ -0,0 +1,62 @@
+require 'net/http'
+
+module MovableInk
+  class AWS
+    module Metadata
+      def http
+        @http ||= begin
+          http = Net::HTTP.new("169.254.169.254", 80)
+          http.open_timeout = 1
```

Comment:
I'm a bit concerned about adding these timeouts at such a low value. If the behavior before was it never times out and the behavior now is it times out fast maybe we can do something halfway for now and use something like 60s.

Reply:
The metadata service is link local.  If it's not connected and responding within a second, I think it's reasonable to expect that it will not respond at all.  In the absence of these short timeouts, using the gem outside of EC2 results in hanging until the timeout before anything responds.